catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/operation/impl/UpdateRequestImpl.java

```diff
@@ -92,6 +92,23 @@ public UpdateRequestImpl(String[] ids, List<Metacard> metacards)
     }
 
     /**
+     * Instantiates a new UpdateRequestImpl from a set of destinations,
+     * an array of ids, and a list of {@link Metacard}
+     *
+     * @param destinations the destination ids this request should be sent to
+     * @param ids          the ids associated with the {@link Metacard} list
```

Comment:
That looks pretty out of scope for this ticket.  I would suggest creating a ticket for that instead.